package-res/lib/protovis.js

```diff
@@ -1,5 +1,5 @@
 /*!
- * Copyright 2002 - 2017 Webdetails, a Hitachi Vantara company.  All rights reserved.
+ * Copyright 2002 - 2018 Webdetails, a Hitachi Vantara company.  All rights reserved.
```

Comment:
Protovis should be edited directly in the https://github.com/webdetails/protovis reposository, in the `transition` branch. Then run `make` and copy the resulting `protovis-d3.3.js` file into CCC, yet renaming it as `protovis.js`. Please issue a PR for  it too.